support-frontend/assets/helpers/abTests/sessionStorage.ts

```diff
@@ -0,0 +1,32 @@
+import * as storage from '../storage/storage';
+import type { Participations } from './models';
+
+// For participation in tests defined in abtestDefinitions.ts
+const PARTICIPATIONS_KEY = 'abParticipations';
+// For participation in landing page tests, which are passed through from the server
+const LANDING_PAGE_PARTICIPATIONS_KEY = 'landingPageParticipations';
+type Key = typeof PARTICIPATIONS_KEY | typeof LANDING_PAGE_PARTICIPATIONS_KEY;
+
+function getSessionParticipations(key: Key): Participations | undefined {
+	const participations = storage.getSession(key);
+	if (participations) {
+		try {
+			return JSON.parse(participations) as Participations;
```

Comment:
fyi: Not necessary but we recently setup a module for session storage, it uses `Valibot.Is` to check objects from session storage match a schema, might be of interest [shown here](https://github.com/guardian/support-frontend/blob/tf-lp-config/support-frontend/assets/pages/%5BcountryGroupId%5D/checkout/helpers/sessionStorage.ts#L31). This is a